app/assets/stylesheets/govuk-component/_govspeak-html-publication.scss

```diff
@@ -109,6 +109,76 @@
         margin-top: ($gutter * 2) + $gutter-two-thirds;
       }
     }
+
+    table.financial-data {
```

Comment:
I could potentially use `.financial-data-table` here instead. But these styles will always be only for tables and having `table.financial-data` makes that clearer and also easier to read. They are here to distinguish from a bare `table` and should be seen as a modifier to it.